packages/demo/test/init.spec.ts

```diff
@@ -50,8 +50,11 @@ describe('Init CLI command', () => {
 
     const ownerAddress = '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266'
 
-    await execAsync(
-      `npx sphinx init --org-id TEST_ORG_ID --sphinx-api-key TEST_SPHINX_KEY --alchemy-api-key TEST_ALCHEMY_KEY --owner ${ownerAddress}`
+    execSync(
+      `export SPHINX_INTERNAL_TEST__SKIP_GIT=true && npx sphinx init --org-id TEST_ORG_ID --sphinx-api-key TEST_SPHINX_KEY --alchemy-api-key TEST_ALCHEMY_KEY --owner ${ownerAddress}`,
```

Comment:
Allows us to skip committing the changes to git during our tests since that could mess with our commit history.